ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -1235,16 +1241,22 @@ objects:
 
         function delete_api_fip() {
           LB_FIP_UID=`cat ${TMP_SHARED}/LB_FIP_UID`
-          echo "Deleting api FIP ... for ${CLUSTER_NAME} with UID: ${LB_FIP_UID}"
+          echo "Deleting api FIP with UID: ${LB_FIP_UID}"
           openstack floating ip delete $LB_FIP_UID
         }
 
         function delete_ingress_fip() {
           INGRESS_FIP_UID=`cat ${TMP_SHARED}/INGRESS_FIP_UID`
-          echo "Deleting ingress FIP using UID: ${INGRESS_FIP_UID}"
+          echo "Deleting ingress FIP with ID: ${INGRESS_FIP_UID}"
           openstack floating ip delete $INGRESS_FIP_UID
         }
 
+        function delete_bootstrap_fip() {
+          BOOTSTRAP_FIP_ID="$(<"${TMP_SHARED}/BOOTSTRAP_FIP_UID")"
```

Comment:
nit: if this is an optimization over the `cat` method used above, perhaps we should also update it in `delete_api_fip` and `delete_ingress_fip` function for consistency?

Reply:
Using builtin redirection instead of forking to a new process for `cat`... OF COURSE it's an improvement. Over the course of the project lifetime, we might save an entire CPU millisecond IF NOT MORE